src/components/data-management/ProjectDetails.jsx

```diff
@@ -723,117 +530,33 @@ const ProjectDetails = ({ width, height }) => {
         visible={analysisModalVisible}
         onLaunch={async (experimentId) => {
           const lastViewed = moment().toISOString();
-          await dispatch(updateExperiment(experimentId, { lastViewed }));
-          await dispatch(updateProject(activeProjectUuid, { lastAnalyzed: lastViewed }));
+          dispatch(updateExperiment(experimentId, { lastViewed }));
+          dispatch(updateProject(activeProjectUuid, { lastAnalyzed: lastViewed }));
```

Comment:
why removing the await?

Reply:
because we don't need it for dispatching actions